frontend/src/interfaces/canvas-context.ts

```diff
@@ -0,0 +1,17 @@
+import { CanvasReducerState } from "@/types/CanvasReducer";
+import { MutableRefObject } from "react";
+import { CanvasMode } from "@/enums/CanvasMode";
+
+export interface ICanvasContext {
```

Comment:
I understand that we couldn't have named it `CanvasContext` (or at least we shouldn't have), but I think that if we decide to name one of the interfaces using the `I`prefix, then we should be consistent and use this naming convention for all of our interfaces.
It's one or the other. Otherwise the names will be confusing

Reply:
Good point, probably i will add I prefix for every interface